futaba/cogs/roles/core.py

```diff
@@ -205,6 +205,39 @@ async def role_remove(self, ctx, *roles: RoleConv):
         content = f"{user_discrim(ctx.author)} removed self-assignable roles: {self.str_roles(roles)}"
         self.journal.send("self/remove", ctx.guild, content, icon="role")
 
+    @role.command(name="createhelperrole", aliases=["chr"])
+    @commands.guild_only()
+    @permissions.check_mod()
+    async def helper_role_add(self, ctx, role: RoleConv, *channels: TextChannelConv):
+        helper_role = self.bot.sql.roles.get_pingable_role_from_original(ctx.guild, role)
```

Comment:
Would add logger info lines for each of these commands.

Reply:
Done.